public/pages/AnomalyCharts/utils/anomalyChartUtils.ts

```diff
@@ -430,3 +453,50 @@ export const getSelectedHeatmapCellPlotData = (
     },
   ] as PlotData[];
 };
+
+export const getAnomalyGradeWording = (isNotSample: boolean | undefined) => {
+  return isNotSample ? 'Anomaly grade' : 'Sample anomaly grade';
+};
```

Comment:
minor: usually prefer the default booleans to have a positive connotation rather than a negative one - like changing `isNotSample` to `isSample`. Not a blocker

Reply:
others like that below as well, feel free to ignore

Reply:
In existing code, `props.showAlerts` == true, then show wording **without** `Sample`, and if it is false or not specified, we show something like `Sample anomaly grade`

We might have to keep using this unless we do more refactoring.

Reply:
Yeah sure, seems fine, just thought I'd throw that out there.